src/phoenix/experiments/functions.py

```diff
@@ -557,23 +579,8 @@ def _get_task_name(task: ExperimentTask) -> str:
     return str(task)
 
 
-def _as_dry_run(obj: RanExperiment, max_sample_size: int = 3) -> RanExperiment:
-    ran_experiment = object.__new__(RanExperiment)
-    if obj.id == DRY_RUN:
-        ran_experiment.__init__(  # type: ignore[misc]
-            **_asdict(obj),
-        )
-    else:
-        dataset = obj.dataset
-        examples = dataset.examples[:max_sample_size]
-        ran_experiment.__init__(  # type: ignore[misc]
-            **{
-                **_asdict(obj),
-                "dataset": replace(dataset, examples=examples),
-                "id": DRY_RUN,
-            }
-        )
-    return ran_experiment
+def _is_dry_run(obj: Any) -> bool:
```

Comment:
Would it make sense to add a boolean `is_dry_run` field to the `RanExperiment` type instead of inferring from the ID?

Reply:
yea, but other things could be dry run too, like evals, or other things. i think this approach is fine (and non-permanent) because the id in this case is invalid.